ortho_seq_code/orthogonal_polynomial.py

```diff
@@ -67,13 +68,22 @@ def orthogonal_polynomial(filename, molecule, phenotype, sites, dm, pop_size, po
     # phi[individual][site][state]. phi[i][j] = vector for site j
     # in individual i.
     DNA_ALPHABETS = ['A', 'C', 'G', 'T']
+    if dm == 4:
```

Comment:
dm could be decided based upon molecule

Reply:
For now, we'll keep these separate because we'll have instances in the future that will make the two non-equivalent. E.g., we could have dm ==4 but a protein grouping that categorizes amino acids into 4 categories (big, small, polar, etc.) so then the molecule would be `protein_grouping_x` or something like that.